packages/e2e-tests/specs/editor/blocks/quote.test.js

```diff
@@ -119,56 +122,6 @@ describe( 'Quote', () => {
 		expect( await getEditedPostContent() ).toMatchSnapshot();
 	} );
 
-	it( 'is transformed to an empty heading if the quote is empty', async () => {
-		await insertBlock( 'Quote' );
-		await transformBlockTo( 'Heading' );
-		expect( await getEditedPostContent() ).toMatchSnapshot();
-	} );
-
-	it( 'is transformed to a heading if the quote just contains one paragraph', async () => {
-		await insertBlock( 'Quote' );
-		await page.keyboard.type( 'one' );
-		await transformBlockTo( 'Heading' );
-		expect( await getEditedPostContent() ).toMatchSnapshot();
-	} );
-
-	it( 'is transformed to a heading and a quote if the quote contains multiple paragraphs', async () => {
-		await insertBlock( 'Quote' );
-		await page.keyboard.type( 'one' );
-		await page.keyboard.press( 'Enter' );
-		await page.keyboard.type( 'two' );
-		await page.keyboard.press( 'Enter' );
-		await page.keyboard.type( 'three' );
-		await transformBlockTo( 'Heading' );
-		expect( await getEditedPostContent() ).toMatchSnapshot();
-	} );
-
-	it( 'is transformed to a heading and a quote if the quote contains a citation', async () => {
-		await insertBlock( 'Quote' );
-		await page.keyboard.type( 'one' );
-		await page.keyboard.press( 'ArrowRight' );
-		await page.keyboard.type( 'cite' );
-		await transformBlockTo( 'Heading' );
-		expect( await getEditedPostContent() ).toMatchSnapshot();
-	} );
-
-	it( 'the resuling quote after transforming to a heading can be transformed again', async () => {
-		await insertBlock( 'Quote' );
-		await page.keyboard.type( 'one' );
-		await page.keyboard.press( 'Enter' );
-		await page.keyboard.type( 'two' );
-		await page.keyboard.press( 'ArrowRight' );
-		await page.keyboard.type( 'cite' );
-		await transformBlockTo( 'Heading' );
-		expect( await getEditedPostContent() ).toMatchSnapshot();
-		await page.click( '[data-type="core/quote"]' );
-		await transformBlockTo( 'Heading' );
-		expect( await getEditedPostContent() ).toMatchSnapshot();
-		await page.click( '[data-type="core/quote"]' );
-		await transformBlockTo( 'Heading' );
-		expect( await getEditedPostContent() ).toMatchSnapshot();
-	} );
-
 	it( 'can be converted to a pullquote', async () => {
```

Comment:
This test is failing because after `ArrowRight` you can't type a `cite`. See https://github.com/WordPress/gutenberg/pull/25892. We need to either fix this keyboard interaction or adapt the test.